src/Behat/Testwork/Output/Printer/ConsoleOutputPrinter.php

```diff
@@ -215,7 +215,7 @@ protected function createOutputStream()
      *
      * @uses createOutputStream()
      */
-    final protected function createOutputConsole($stream = null)
+    protected function createOutput($stream = null)
```

Comment:
A better way to be able to extend this point.

I'd move the whole `createOutput()` or `createOutputConsole()` behaviour to the external Factory instead of relying on override of protected methods :)